src/plugins/data_source_management/public/components/text_content/text_content.ts

```diff
@@ -6,278 +6,258 @@
   {
     defaultMessage: 'Credential Source',
   }
 );
 
 /* Edit data source */
-export const dataSourceNotFound = i18n.translate(
+export const DATA_SOURCE_NOT_FOUND = i18n.translate(
   'dataSourcesManagement.editDataSource.dataSourceNotFound',
   {
     defaultMessage: 'Data Source not found!',
   }
 );
-export const deleteThisDataSource = i18n.translate(
+export const DELETE_THIS_DATA_SOURCE = i18n.translate(
   'dataSourcesManagement.editDataSource.deleteThisDataSource',
   {
     defaultMessage: 'Delete this Data Source',
   }
 );
-export const oldPasswordText = i18n.translate('dataSourcesManagement.editDataSource.oldPassword', {
-  defaultMessage: 'Old password',
-});
-export const newPasswordText = i18n.translate('dataSourcesManagement.editDataSource.newPassword', {
-  defaultMessage: 'New password',
-});
-export const confirmNewPasswordText = i18n.translate(
-  'dataSourcesManagement.editDataSource.confirmNewPassword',
+export const NEW_PASSWORD_TEXT = i18n.translate(
+  'dataSourcesManagement.editDataSource.newPassword',
   {
-    defaultMessage: 'Confirm new password',
+    defaultMessage: 'New password',
   }
 );
-export const updatePasswordText = i18n.translate(
-  'dataSourcesManagement.editDataSource.updatePasswordText',
+export const UPDATE_STORED_PASSWORD = i18n.translate(
+  'dataSourcesManagement.editDataSource.updateStoredPassword',
   {
-    defaultMessage: 'Update password',
+    defaultMessage: 'Update stored password',
   }
 );
-export const connectionDetailsText = i18n.translate(
+export const CONNECTION_DETAILS_TITLE = i18n.translate(
   'dataSourcesManagement.editDataSource.connectionDetailsText',
   {
     defaultMessage: 'Connection Details',
   }
 );
-export const objectDetailsText = i18n.translate(
+export const OBJECT_DETAILS_TITLE = i18n.translate(
   'dataSourcesManagement.editDataSource.objectDetailsText',
   {
     defaultMessage: 'Object Details',
   }
 );
-export const objectDetailsDescription = i18n.translate(
+export const OBJECT_DETAILS_DESCRIPTION = i18n.translate(
   'dataSourcesManagement.editDataSource.objectDetailsDescription',
   {
     defaultMessage:
       'This connection information is used for reference in tables and when adding to a data source connection',
   }
 );
-export const authenticationMethodTitle = i18n.translate(
-  'dataSourcesManagement.editDataSource.authenticationMethodTitle',
-  {
-    defaultMessage: 'Authentication Method',
-  }
-);
-export const authenticationTitle = i18n.translate(
+export const CREDENTIAL = i18n.translate('dataSourcesManagement.editDataSource.credential', {
+  defaultMessage: 'Credential',
+});
+export const AUTHENTICATION_TITLE = i18n.translate(
   'dataSourcesManagement.editDataSource.authenticationTitle',
   {
     defaultMessage: 'Authentication',
   }
 );
-export const authenticationDetailsText = i18n.translate(
-  'dataSourcesManagement.editDataSource.authenticationDetailsText',
+export const AUTHENTICATION_METHOD = i18n.translate(
```

Comment:
didn't find any usage of this const

Reply:
it is used in edit_data_source_form line 410

Reply:
<img width="754" alt="Screen Shot 2022-10-05 at 7 00 18 PM" src="https://user-images.githubusercontent.com/109986843/194197610-e4e7d817-33cb-40df-b0f8-dc1a1afe83ed.png">